include/fmt/core.h

```diff
@@ -1341,26 +1341,20 @@ typename std::enable_if<
   return out;
 }
 
-template <typename Container, typename... Args>
-inline typename std::enable_if<
-  is_contiguous<Container>::value, std::back_insert_iterator<Container>>::type
-    format_to(std::back_insert_iterator<Container> out,
-              string_view format_str, const Args &... args) {
-  format_arg_store<format_context, Args...> as{args...};
-  return vformat_to(out, format_str, as);
-}
-
-template <typename Container, typename... Args>
+#define FMT_CHAR(Str) typename internal::format_string_traits<Str>::char_type
```

Comment:
Please add a blank line to separate the macro from the function definition.

Reply:
The macro is there to support gcc versions older than 4.7, e.g. gcc 4.6.4 released in 2013. I don't like it either and eventually it will be replace with the type alias you wrote or similar.

It looks like `FMT_CHAR` is defined twice now.

Reply:
Oops, too much git history editing without coffee. Fixed now.